### DISCUSSION TITLE
Error: Text Generator Plugin

### DISCUSSION THREAD

**Discussion by CharlyDN** [nhaouari/obsidian-textgenerator-plugin#88] [Q&A]
Hello everyone, I just installed this Plugin and although I have everything configured correctly and the GPT account created and the API Key generated, it gives me the following error: "Error: Text Generator Plugin: Error Check Console CTRL+SHIFT+I"
I need help to solve it, I remain attentive and grateful, regards

**Comment by rasulamirov** [comment]
I have the same issue.

**Comment by ak-9701-d** [comment]
i have the same issue

**Comment by cgofun** [comment]
i have the same issue
